ods_ci/tests/Tests/0600__distributed_workloads/0602__training/test-smoke.robot

```diff
@@ -28,11 +28,21 @@ Ray smoke test
         FAIL    Can not find kuberay-operator service in ${APPLICATIONS_NAMESPACE}
     END
     Log To Console    kuberay-operator service exists
-    Log To Console    Verifying kuberay-operator's container image is referred from registry.redhat.io
-    ${pod} =    Find First Pod By Name  namespace=${APPLICATIONS_NAMESPACE}   pod_regex=kuberay-operator-
-    Container Image Url Should Contain      ${APPLICATIONS_NAMESPACE}     ${pod}      kuberay-operator
-    ...     registry.redhat.io/rhoai/odh-kuberay-operator-controller
-    Log To Console    kuberay-operator's container image is verified
+    ${test_env}=  Is Test Enviroment ROSA-HCP
+    IF    ${test_env}==True
+        # We use Kyverno custom policies to pull unreleased images from quay registry for hypershift clusters
```

Comment:
duplicate code; extract to a keyword and re-use

Reply:
Updated the code @rnetser, please review